src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -310,12 +310,24 @@ object GenericClient {
     asJsonEither[CdpApiError, T].mapWithMetadata((response, metadata) =>
       response match {
         case Left(DeserializationException(_, error)) =>
```

Comment:
Nicer with `case Left(DeserializationException(_, error)) if metadata.code == StatusCode.TooManyRequests =>` and letting the old `case` remain as is (but after the new one, of course).

Reply:
sure